packages/mrl/src/getTransferData/getMoonChainData.ts

```diff
@@ -37,9 +37,11 @@ export async function getMoonChainData({
   }
 
   const fee = await getDestinationFee({
-    asset,
+    address: sourceAddress, // TODO not correct
```

Comment:
What's that?

Reply:
I needed to pass the address in this function but since this is from MRL I decided to do it later. However I will review the interface from your other comment